charts/ratify/templates/deployment.yaml

```diff
@@ -165,13 +165,6 @@ spec:
         - name: tls
           secret:
             secretName: {{ include "ratify.fullname" . }}-tls
```

Comment:
Do we need to mount the secret if the cert rotator is enabled? Does the cert rotator handle the mounting on its own?

Reply:
yeah, we need to mount an empty secret in this case